planning/scenario_planning/parking/astar_search/include/astar_search/astar_search.h

```diff
@@ -17,17 +17,18 @@
 #ifndef ASTER_PLANNER_H
```

Comment:
[Minor]: There is a typo here - not sure if its important for this PR though.

Reply:
I fixed. Also added `SCENARIO_PLANNING_` prefix.